app/bundles/PluginBundle/Translations/en_US/validators.ini

```
@@ -1 +1,2 @@
-mautic.plugin.field.required_mapping_missing="At least one required field is not mapped."
\ No newline at end of file
+mautic.plugin.field.required_mapping_missing="At least one required field is not mapped."
+mautic.lead_list.not_allowed_plugin_publish="You are not allowed to publish plugin due to insufficient configurations."
```

It doesn't sound right. Should there be `the` or `this`?
```suggestion
mautic.lead_list.not_allowed_plugin_publish="You are not allowed to publish this plugin due to insufficient configurations."
```